src/Icon/README.md

```diff
@@ -1,21 +1,13 @@
 # Icon
 
-The Icon component is actually an object which holds all Icon components. 
-Trying to render `<Icon />` will result in an error, but for example, `<Icon.ArrowBack />` would not.
+All icons are rendered individually by their name, i.e. **`ArrowBack`** or**`Expand`**. 
 
 ## Usage
-    import { Icon } from '@retailmenot/anchor';
```

Comment:
Importing the icons in this way will import ALL of the icons into a bundle, I believe. We don't have an example of that since that's what we're trying to avoid. We're doing it here so that we can show all of them.